README.md

```diff
@@ -37,6 +37,13 @@ to the RDO project.
 * Add your name and IRC nick to the [mentor list](#mentors)
 * Follow the steps in [I want to add an easy fix issue](#i-want-to-add-an-easyfix-issue)
 
+## Checklist for sending your first RDO gerrit Review:
+* [ ] Add the changed files using git add command
+* [ ] In the commit message: 
+      * First line: What you have fixed?
+      * Second Line: Fixed in <RDO easy Fix issue link>
+* [ ] Save the commit message and then send review using git review.
```

Comment:
I would add instructions in case the person needs to send an edit (see http://eavesdrop.openstack.org/irclogs/%23rdo/%23rdo.2017-07-13.log.html#t2017-07-13T15:17:21 )